apps/admin-ui/components/Layout/Header/Breadcrumbs.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Breadcrumbs, Link, Typography } from "@mui/material";
+import { BreadcrumbsNameMaps } from "./Header";
+import { useRouter } from "next/router";
+
+interface BreadcrumbProps {
+    breadcrumbs: BreadcrumbsNameMaps
+}
+
+export default function Breadcrumb({ breadcrumbs: {
+    title,
+    links
+} }: BreadcrumbProps) {
+    const { pathname } = useRouter()
+    const pathnames = pathname.split('/').filter((x) => x)
+    console.log(pathname.split('/'), pathnames)
+    return (
+        <Box sx={{
+            display: 'grid',
+            textAlign: 'start'
+        }}>
+            <Typography
+                variant="h1"
+                sx={{
+                    fontSize: '36px',
+                    lineHeight: '44px',
+                    letterSpacing: '-0.02px',
+                    padding: 0
+                }}
+            >{title}</Typography>
+            <Breadcrumbs>
+                <Link
+                    variant="body2"
+                    underline="hover"
+                    color="inherit"
+                >
+                    Management
```

Comment:
this text should be translated
and I'm not sure it'll appear in all routes,

think about it and see if it should not be gotten rid of

Reply:
Okey!